pkg/kube/config.go

```diff
@@ -19,10 +19,11 @@ package kube // import "helm.sh/helm/v3/pkg/kube"
 import "k8s.io/cli-runtime/pkg/genericclioptions"
 
 // GetConfig returns a Kubernetes client config.
-func GetConfig(kubeconfig, context, namespace string) *genericclioptions.ConfigFlags {
+func GetConfig(kubeconfig, context, token, namespace string) *genericclioptions.ConfigFlags {
```

Comment:
Users consuming `kube.GetConfig` would have to change their function signatures once they upgrade. Unfortunately, this would be considered a backwards-incompatible change, so we cannot accept it as it currently stands.

One way to work around this would be to revert this change, and just make `RestClientGetter` set the token parameter directly. Would you mind making that change?

Reply:
@bacongobbler thanks for checking this out I will make that change then.

Reply:
@bacongobbler I have made the necessary changes to check it out